### PR TITLE
Fix for issue https://github.com/yonaskolb/XcodeGen/issues/392.

### PR DESCRIPTION
**Pull request by vgorloff**

### CODE REVIEW

File: Sources/ProjectSpec/SpecValidation.swift [hunk 1]
```diff
@@ -135,7 +135,7 @@ extension Project {
 
         for scheme in schemes {
             for buildTarget in scheme.build.targets {
-                if getTarget(buildTarget.target) == nil {
+                if getProjectTarget(buildTarget.target) == nil {
```
Comment (vgorloff) [inline Sources/ProjectSpec/SpecValidation.swift]: Fixes Validation error.

File: Sources/XcodeGenKit/ProjectGenerator.swift [hunk 2]
```diff
@@ -41,15 +41,25 @@ public class ProjectGenerator {
...
-            let target = project.getTarget(buildTarget.target)!
-            let buildableReference = XCScheme.BuildableReference(
-                referencedContainer: "container:\(project.name).xcodeproj",
-                blueprintIdentifier: targetReference.reference,
-                buildableName: target.filename,
-                blueprintName: buildTarget.target
-            )
-
-            return XCScheme.BuildAction.Entry(buildableReference: buildableReference, buildFor: buildTarget.buildTypes)
+            if let target = project.getTarget(buildTarget.target) {
```
Comment (vgorloff) [inline Sources/XcodeGenKit/ProjectGenerator.swift]: Fixes crash on explicitly unwrapped optional.

Comment (yonaskolb) [reply Sources/XcodeGenKit/ProjectGenerator.swift]: Nice. Could we just get the `buildableName` in these `ifs`, and then keep the common code below those?

**Comment by yonaskolb** [comment]
Thank you @vgorloff! One small change, and then we can get this merged 👍

File: Sources/XcodeGenKit/ProjectGenerator.swift (hunk 2 above)
Comment (vgorloff) [reply Sources/XcodeGenKit/ProjectGenerator.swift]: Ah. I see. Copy paste code. Will fix it today later.

**Pull request merged**